### PR TITLE
Tighten up a bunch of test accurasy requirements now that have FiniteDifferences v0.11.4

### PR DESCRIPTION
**Pull request by oxinabox**
Closes https://github.com/JuliaDiff/ChainRules.jl/issues/284
by undoing the changes made in 
https://github.com/JuliaDiff/ChainRules.jl/commit/9fc8c57055b7e3315b747ea648336fb5f75bdca8#diff-18f60c608d1cadf3013a4e16d696c43ea9f488fa49caa48cb6b582481025bc0fR57
(well at least the once that required changing the `atol`/`rtol`, i figure the others can stay)

requires https://github.com/JuliaDiff/FiniteDifferences.jl/pull/125

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -6,6 +6,7 @@ version = "0.7.33"
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 ChainRulesTestUtils = "cdddcdb0-9152-4a09-a978-84456f9df70a"
 Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+FiniteDifferences = "26cc04aa-876d-5657-8c51-4c34ba976000"
```
Comment (simeonschaub) [inline Project.toml:9]: That doesn't look right. I don't think we want to add FiniteDifferences as a dependency?

Comment (oxinabox) [reply Project.toml:9]: Good catch

File: test/rulesets/Base/fastmath_able.jl [hunk 2]
```diff
@@ -142,8 +140,8 @@ const FASTABLE_AST = quote
             Δz = randn(typeof(f(x, y)))
 
             # some tests struggle in fast_math mode to get accurasy so we relax it some.
-            frule_test(f, (x, Δx), (y, Δy); rtol=(is_fastmath_mode ? 1e-5 : 1e-7))
-            rrule_test(f, Δz, (x, x̄), (y, ȳ); rtol=(is_fastmath_mode ? 1e-5 : 1e-7))
+            frule_test(f, (x, Δx), (y, Δy))
```
Comment (nickrobinson251) [inline test/rulesets/Base/fastmath_able.jl:143]: can we delete comment above too? (github doesn't seem to let me directly suggest that change)

**Review by nickrobinson251 — APPROVED** [review]

**Comment by oxinabox** [comment]
Weird that this was passing without  JuliaDiff/FiniteDifferences.jl#125
being merged.
May have to revert if CI was bugging out somehow?

**Comment by willtebbutt** [comment]
CI isn't currently functional, because travis. I believe the only test that was running was the Zygote integration GHA

**Pull request merged**